contrib/demo-windows.md

```diff
@@ -0,0 +1,85 @@
+## Requirement
+Here is the quick instruction for anyone who wants to check out the remote execution POC (proof of concept) in the super agent.
+
+You don't need to have knowledge of the techonology behind super agent. But you need to have basic knowledge of Microsoft Remote Desktop, Powershell command in Windows and know how to use a browser.
+
+## Connect to windows server and set up Chrome browser
+
+Use Microsoft Remote Desktop to connect to the windows server.  If you are using Mac, you need to install the Remote Desktop here: https://itunes.apple.com/us/app/microsoft-remote-desktop/id715768417?mt=12
```

Comment:
I thought that was the official way to download it? I actually don't trust (neither dose my Mac) anything not downloaded from outside AppStore.